azext_iot/tests/iothub/core/test_iothub_storage_int.py

```diff
@@ -247,15 +230,15 @@ def test_system_identity_storage(self):
                 self.check_for_running_import_export()
 
                 job_id = self.cmd(
-                    'iot hub device-identity export -n {} --bcu "{}" --auth-type {} --identity {} --ik true'.format(
-                        self.entity_name, self.live_storage_uri, "identity", "[system]"
+                    'iot hub device-identity export -n {} --bcu "{}" --ik true'.format(
```

Comment:
Isn't this meant to be a system managed identity based export? If so isn't `--identity [system]` required?

Reply:
Yeah that was unintended, I have fixed it now.